README.md

```diff
@@ -250,7 +141,7 @@ Feel free to check the [issues page](../../issues/).
 
 > Write a message to encourage readers to support your project
```

Comment:
- Hi :clap:, this placeholder text was added to guide you on making updates to the readme to suit your needs.  kindly remove it as it's not meant to be in the readme after you've updated it. :pray: